lib/gameComponents/deck.dart

```diff
@@ -1 +1,30 @@
-class Deck {}
+import 'card.dart';
+import 'hand.dart';
+
+class Deck {
+  List<Hand> hands = [];
```

Comment:
💭 Should the deck have knowledge of the hands, or should this be handled by another class such as `Game`?